### PR TITLE
Create script for rebase in one step

### PR DESCRIPTION
**Pull request by sebmancipe**
Create a new script to avoid doing several steps when rebasing from a branch

### CODE REVIEW

File: html-practices/my-new-change/script.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    exit 1
+fi
+
+current=$(git branch --show-current)
+target=$1
+
+git checkout $target
+git pull
+git checkout $current
+git rebase $target
```
Comment (sebmancipe) [inline html-practices/my-new-change/script.sh:14]: Please set a new line at the end of the file

File: html-practices/my-new-change/script.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ $# -eq 0 ]; then
+    echo "No target branch name provided, command ignored"
+    exit 1
+fi
+
```
Comment (Crokwar) [inline html-practices/my-new-change/script.sh:7]: Simon si puedo escribir

File: html-practices/my-new-change/script.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```
Comment (sebmancipe) [inline html-practices/my-new-change/script.sh:2]: **chore** 🧹
Please set a comment to explain the purpose of this file

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>

File: html-practices/my-new-change/script.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+if [ $# -eq 0 ]; then
+    echo "No target branch name provided, command ignored"
+    exit 1
+fi
```
Comment (sebmancipe) [inline html-practices/my-new-change/script.sh:6]: **suggestion** 💭
There's a better way to do this:
```suggestion
    echo "No target branch name provided, command ignored"
    exit 0
fi
```

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>

**Review by Crokwar — APPROVED** [review]

File: html-practices/my-new-change/script.sh (hunk 1 above)
Comment (HarleyC0) [reply html-practices/my-new-change/script.sh:14]: falta motivacion Tstststs

**Pull request merged**